chain-storage-sqlite/examples/import.rs

```diff
@@ -0,0 +1,92 @@
+extern crate cardano_storage;
+extern crate chain_storage;
+extern crate cardano;
+
+use chain_core::property::Block;
+use cardano_storage::{StorageConfig};
+use chain_storage::store::{BlockStore};
+use std::time::{Instant};
+use std::str::FromStr;
+
+fn main() {
+    let storage_config = StorageConfig::new(&"/home/eelco/.local/share/cardano-cli/blockchains/staging".into());
+    let storage = cardano_storage::Storage::init(&storage_config).unwrap();
+
+    let genesis_hash = cardano::block::HeaderHash::from_str(&"c6a004d3d178f600cd8caa10abbebe1549bef878f0665aea2903472d5abf7323").unwrap();
+
+    let first_hash = cardano::block::HeaderHash::from_str(&"b365f1be6863b453f12b93e1810909b10c79a95ee44bf53414888513fe172c90").unwrap();
+
+    let tip_hash = cardano::block::HeaderHash::from_str(&"159e3cfe147dfbe302daf37b63dbce2e676cd88c1419f693ad9ebdc69cf4bc1c").unwrap();
+
+    let mut store = chain_storage_sqlite::SQLiteBlockStore::new(
+        genesis_hash, "/tmp/chain.sqlite");
```

Comment:
this path seems a bit inflexible and inherently temporary , shouldn't it use chain-storage variables that where defined in the cargo.toml ?